### PR TITLE
Finish updating strict data with macrons

### PR DESCRIPTION
**Pull request by dvr333**

### CODE REVIEW

File: db/data-strict.txt [hunk 1]
```diff
@@ -5,235 +5,235 @@ Gate (feminine): porta portae portae portam portā
...
 
 --- Future Perfect ---
-Praise: laudavero laudaveris laudaverit laudaverimus laudaveritis laudaverint
-Remind: monuero monueris monuerit monuerimus monueritis monuerint
-Lead: duxero duxeris duxerit duxerimus duxeritis duxerint
-Hear: audivero audiveris audiverit audiverimus audiveritis audiverint
-Take: cepero ceperis ceperit ceperimus ceperitis ceperint
+Praise: laudāverō laudāveris laudāverit laudāverimus laudāveritis laudāverint
+Remind: monuerō monueris monuerit monuerimus monueritis monuerint
+Lead: dūxero dūxeris dūxerit dūxerimus dūxeritis dūxerint
```
Comment (coyotespike) [inline db/data-strict.txt]: missed one, dūxerō :)

File: db/data-strict.txt [hunk 2]
```diff
@@ -5,235 +5,235 @@ Gate (feminine): porta portae portae portam portā
...
-Praise: laudaverim laudaveris laudaverit laudaverimus laudaveritis laudaverint
-Remind: monuerim monueris monuerit monuerimus monueritis monuerint
-Lead: duxerim duxeris duxerit duxerimus duxeritis duxerint
-Hear: audiverim audiveris audiverit audiverimus audiveritis audiverint
-Take: ceperim ceperis ceperit ceperimus ceperitis ceperint
+Praise: laudāverim laudāverīs laudāverit laudāverīmus laudāverītis laudāverint
+Remind: monuerim monuerīs monuerit monuerīmus monuerītis monuerint
+Lead: dūxerim dūxerīs dūxerit dūxerīmus dūxerītis dūxerint
+Hear: audīverim audīverīs audīverit audīverīmus audīverītis audīverint
+Take: cēperim cēperīs cēperit cēperīmus cēperītis cēperint
```
Comment (coyotespike) [inline db/data-strict.txt]: The sources I've looked at show no macrons on the i's, so to speak. Hence all of these become like: `laudāverim laudāveris laudāverit laudāverimus laudāveritis laudāverint`

Thoughts?

**Review by coyotespike — COMMENTED** [review]
Many thanks to @JonathanAquino for making this website and starting to add macrons! I enabled the cookie in devtools and it works great.

And big kudos to @dvr333 for knocking out the rest of the macrons!

File: db/data-strict.txt (hunk 2 above)
Comment (dvr333) [reply db/data-strict.txt]: You’re right, I got a bit overenthusiastic about the tū, nōs, and vōs conjugation macrons for past perfect, but future perfect lacks them, as you say

File: db/data-strict.txt (hunk 1 above)
Comment (dvr333) [reply db/data-strict.txt]: Fixed

**Comment by JonathanAquino** [comment]
Thank you!

**Pull request merged**